tests/settings/SettingTest.py

```diff
@@ -14,9 +14,12 @@ class SettingTest(unittest.TestCase):
     def test_construction(self):
         self.assertRaises(ValueError, Setting, '', 2, 2)
         self.assertRaises(TypeError, Setting, '', '', '', from_cli=5)
+        self.assertRaisesRegex(TypeError, 'to_append.*',
+                               Setting, '', '', '', to_append=10)
 
     def test_path(self):
-        self.uut = Setting('key', ' 22\n', '.' + os.path.sep, True)
+        self.uut = Setting(
+            'key', ' 22\n', '.' + os.path.sep, strip_whitespaces=True)
```

Comment:
agreed, I'd personally be willing to let this slip through to get this merged. Big PRs should be avoided before they happen but it's too late for that now and we gotta get this finished.